### PR TITLE
Ommer footnote, link words to maths, Transaction_Receipt

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

**Comment by pirapira** [comment]
Please fix the Travis error.

File: Paper.tex [hunk 1]
```diff
@@ -321,7 +321,7 @@ \subsection{The Transaction} \label{ch:transaction}
...
+The block in Ethereum is the collection of relevant pieces of information (known as the block \textit{header}), $H$, together with information corresponding to the comprised transactions, $\mathbf{T}$,\hypertarget{ommerheaders}{} and a set of other block headers $\mathbf{U}$ that are known to have a parent equal to the present block's parent's parent (such blocks are known as \textit{ommers}\footnote{\textit{ommer} is the most prevalent (which is not saying much, in that it isn't commonly used) gender-neutral term to mean ``sibling of parent''; see \url{https://nonbinary.miraheze.org/wiki/Gender_neutral_language\#Aunt.2FUncle}}). The block header contains several pieces of information:
```
Comment (pirapira) [inline Paper.tex]: I find this too verbose.

"in that it isn't commonly used" explaining "which is not saying much" annotating "the most prevalent" in the foot note about "ommer".

Comment (jamesray1) [reply Paper.tex]: I don't know if you get notifications after I make a change, so I'm replying here so that you'll get a notification.

**Comment by pirapira** [comment]
(not to say it's common)

**Comment by ldct** [comment]
I think we can shorten to 

``omer" is a gender neutral term to refer to the sibling of a parent \cite

and the interested reader can read the cited text

**Pull request merged**